run_pipeline_update.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+#SBATCH -J sm
+#SBATCH -o out
+#SBATCH -e err
+#SBATCH -p shared
+#SBATCH -n 1
+#SBATCH -t 9000
+#SBATCH --mem=10000
+
+CONDA_BASE=$(conda info --base)
+source $CONDA_BASE/etc/profile.d/conda.sh
+conda activate snakemake
+
+#this forces jobs with updated parameters to rerun
+snakemake -R $(snakemake --list-params-changes) --snakefile workflow/Snakefile --profile ./profiles/slurm
```

Comment:
This probably needs to be $(snakemake --list-input-changes)?

Reply:
I've tested this and --list-input-changes doesn't always seem to work, I guess because we are not actually adding new input files, we are just changing the sample sheet, but this may be a subtle bug in our code somewhere (it throws a 'no rule found' error in at least some cases). --list-params-changes has the side effect that if you change anything else in the config it will rerun those modified rules as well, but ideally with the new interval code this will have a minimal impact. 

Realistically a user should probably check what will be rerun using both --list-params-changes and --list-input-changes and pick what works for their use case.